styles/badge.css

```diff
@@ -0,0 +1,59 @@
+@import url(/styles/docs.css);
+@import url(/styles/index.css);
+
+.icon-button {
+  position: relative;
+  width: 5rem;
```

Comment:
You can define and use variables instead of absolute values

Reply:
It is a big value which probably wouldn't be used in the future, for all the common values, I'll be using variables instead.
Thanks for the suggestion